lm-agent/lm_agent/server_interfaces/dsls.py

```diff
@@ -0,0 +1,70 @@
+"""DSLS license server interface."""
+import typing
+
+from lm_agent.models import LicenseServerSchema, LicenseReportItem
+from lm_agent.config import settings
+from lm_agent.exceptions import LicenseManagerBadServerOutput
+from lm_agent.server_interfaces.license_server_interface import LicenseServerInterface
+from lm_agent.parsing import dsls
+from lm_agent.utils import run_command
+
+
+class DSLSLicenseServer(LicenseServerInterface):
+    """Extract license information from DSLS license server."""
```

Comment:
it would be good if we had an example of the raw license server information to understand what kind of information we want to extract

Reply:
The output examples are in the `conftest`, it's a very big text file that would clutter the module if added here.